lib/src/view/engine/engine_lines.dart

```diff
@@ -0,0 +1,174 @@
+import 'package:collection/collection.dart';
+import 'package:dartchess/dartchess.dart';
+import 'package:fast_immutable_collections/fast_immutable_collections.dart';
+import 'package:flutter/material.dart';
+import 'package:flutter_riverpod/flutter_riverpod.dart';
+import 'package:lichess_mobile/src/constants.dart';
+import 'package:lichess_mobile/src/model/account/account_preferences.dart';
+import 'package:lichess_mobile/src/model/analysis/analysis_preferences.dart';
+import 'package:lichess_mobile/src/model/common/eval.dart';
+import 'package:lichess_mobile/src/model/engine/evaluation_service.dart';
+import 'package:lichess_mobile/src/model/settings/brightness.dart';
+import 'package:lichess_mobile/src/view/engine/engine_gauge.dart';
+import 'package:lichess_mobile/src/widgets/buttons.dart';
+
+class EngineLines extends ConsumerWidget {
+  const EngineLines({
+    required this.onTapMove,
+    required this.clientEval,
+    required this.isGameOver,
+    required this.isLandscape,
+  });
+  final void Function(NormalMove move) onTapMove;
+  final ClientEval? clientEval;
+  final bool isGameOver;
+  final bool isLandscape;
```

Comment:
Since it is now a public widget, I'd use an enum in place of that boolean. We already have that one: https://github.com/lichess-org/mobile/blob/ef9fc55fdc08a606f26a7069140e73a1308c3e21/lib/src/view/engine/engine_gauge.dart#L20

Reply:
Hmm, the only thing it affects is the padding, so maybe it's better to simply pass the padding as a parameter instead?

Reply:
Removed the `isLandscape` and `Padding` entirely, the calling code should decide padding instead